content/notifications/send-notifications/get-started/user-guide/_index.en.md

```diff
@@ -4,50 +4,9 @@ linktitle: User guide
 weight: 10
 description: "A user guide for the notification service"
 ---
-
-{{% notice info %}}
```

Comment:
I suggest we keep all the old info, but simply add this SMS section.
Keeping the commented out part of things as well as a starting point for Torkel